module/VuFind/src/VuFind/Db/Entity/User.php

```diff
@@ -158,7 +158,7 @@ class User implements UserEntityInterface
      *
      * @var ?string
      *
-     * @ORM\Column(name="cat_username", type="string", length=50, nullable=true)
+     * @ORM\Column(name="catUsername", type="string", length=50, nullable=true)
```

Comment:
Doesn't the `name` property here refer to the actual column name in the database? If so, that should remain `cat_username` (and `cat_password` below), not `catUsername` / `catPassword`. Of course, if I'm mistaken and the framework does some kind of automatic normalization/matching, please feel free to correct me!